modules/experimental/pages/portal.adoc

```diff
@@ -0,0 +1,98 @@
+* `_(4+)_` allows Vehicle units that have an Armour value of `4+` or lower to use the {portal}.
+* `_(Walker)_` allows Vehicle units that have the {walker} ability to use the {portal}, regardless of their Armour value.
+
+We also indicate how the portal affects the arrival of detachments that use it:
+
+* `_(Auto)_` means that detachments in Portal Reserves do *not* follow the standard rules for Reserves.
+Instead, from turn two onwards, you must *must* disembark them from a {portal} unit/objective in the Movement phase unless there are not enough portals available, or none of the available portals can disembark all of the units in the detachment.
+* `_(Roll)_` means that detachments in Portal Reserves follow the standard rules for Reserves.
+When a detachment arrives from Portal Reserves, you *must* disembark it from a {portal} unit/objective, if possible.
+
+== Game effects of {portal}
+
+Where your army includes any units or objectives that have the {portal} ability:
+
+* Regardless of the scenario, you may keep any detachments that can use a portal out of the play area, as <<Portal Reserves>>.
+
+Where your unit or objective has the {portal} ability:
+
+* When you wish to move the units in a detachment out of a portal, use the normal disembarkation rules as at xref:main-rules:transportation.adoc[].
+footnote:[
+When you disembark units from an *objective* that has the {portal} ability, you obviously cannot pay 5 cm of its movement like you would with a mobile transport unit.
+In this case, just deduct 5 cm from the movement of each unit that you disembark, as usual, and ignore the deduction that would normally apply to the transport unit.
+]
+* When you wish to embark or disembark a detachment through a portal, you may choose only Advance or Assault orders.
+* You may use each individual portal only *once per turn*, to embark or disembark a single detachment.
+* In the Movement phase, you may embark friendly units 'into' a portal.
+But then you *must* immediately disembark them from a different {portal} elsewhere in the play area -- you may not place them outside of the play area again, so they cannot become Portal Reserves again.
+
+== Portal Reserves
+
+Apply these rules to each detachment that you have outside of the play area as Portal Reserves.
+
+When the detachment should arrive, you must put it into play if possible -- but you may only put it into play through a suitable, available {portal}.
+
+* If there is no portal available when you need it, then you must leave the detachment in Portal Reserves this turn -- it does not arrive yet after all.
+You must roll again next turn.
+* If none of the available portals can disembark all of the units in the detachment, then you may choose, either:
+** Leave the detachment in Portal Reserves this turn -- it does not arrive yet after all.
+You must roll again next turn.
+** Disembark as much of the detachment as you can.
+The units that cannot disembark are immediately destroyed.
```

Comment:
This seems a bit strange to me:

First, it seems illogical that units that didn't participate in the battle are automatically _destroyed_ just because they are 'running late'. 'Count as destroyed' for some game purposes would be one thing, but to imply 'are actually destroyed as a consequence of not being on time to participate in the battle' is another.

Second, I wonder what you intend for Army Morale -- if all portals are destroyed, and thus all remaining Portal Reserves are destroyed, I believe this implies that we must deduct their Morale points at the end of the next Rally phase -- is this correct/what you intend?

Reply:
Yes, I intend that all detachments still in Portal Reserves are destroyed, with all the usual effects on Army Morale. It's a very powerful ability, with a very powerful downside.

I don't mind whether we do that by saying that the detachments 'are' destroyed, or by saying that they count as destroyed for all gaming purposes.

Reply:
Okay, makes sense.

One quibble: While they count as destroyed for all in-game 'battle' purposes, might it be wise to phrase in such a way that it's clear that they aren't 'destroyed/wiped out' for campaign 'war' purposes?

I'm thinking of the rules at [Are you experienced?](https://thehobby.zone/resources/e40k-compendium/Content/Battles/Campaigns/AreYouExperienced.htm), especially under 'Being wiped out' and 'Replacing casualties'.

Reply:
Fine by me.

Reply:
Perhaps something like this. Might be room to streamline it some more.
```suggestion
The units that cannot disembark count as destroyed for the purposes of this battle.
footnote:[
For the purposes of an ongoing campaign, we do *not* consider these units to be destroyed.
This means that if you use rules like link:http://https://thehobby.zone/resources/e40k-compendium/Content/Battles/Campaigns/AreYouExperienced.htm[Are you experienced?^] (external URL) then their detachment is not 'wiped out'.
But as far as morale and victory conditions for the current battle are concerned, treat them  as destroyed.
]
```

Reply:
Streamlining suggestion:
```suggestion
The units that cannot disembark count as destroyed for the purposes of army morale and victory conditions.
]
```
That hopefully removes the need for an explanatory footnote.

Reply:
Yeah, that seems good — I started off with the idea to leave it somewhat broad with "purposes of this battle" in case there was stuff I hadn't thought of, as well as the idea to reference the existing campaign/experience rules. But, if you're happy that this boils down to just army morale and victory conditions and there is nothing else, then your more minimal suggestion works and I can live without the reference to those other rules.

Reply:
A tiny tweak to your suggestion, just so that we don't insert a redundant square bracket.
```suggestion
The units that cannot disembark count as destroyed for the purposes of army morale and victory conditions.
```